File: Mobile/codingkids/ms-authentification/src/main/java/com/pfe/msauthentification/Model/EnfantLearning.java
package com.pfe.msauthentification.Model;

import com.mongodb.lang.Nullable;
import lombok.Data;

@Data

public class EnfantLearning {
    private String nom;
    private int points;
    private int points2;
    private long   points_sequencing_blockly;
    private long  points_loop_blockly;
    private long  points_condition_blockly;

}

File: app/src/app/enfantLearning.ts
import { Level } from './level';

// tslint:disable-next-line:class-name
export class enfantLearning {
  id!: string;
  nom!: string;
  points!: number;
  points2!: number;
  // tslint:disable-next-line:variable-name
  points_sequencing_blockly!: number;
  // tslint:disable-next-line:variable-name
  points_loop_blockly!: number;
  // tslint:disable-next-line:variable-name
  points_condition_blockly!: number;

  level!: Level[];
}
